docs/config.rst

```diff
@@ -313,6 +313,22 @@ Produces these sessions when running ``nox --list``:
     * tests(mysql, new)
 
 
+Minimum required nox version
+----------------------------
+
+Minimum required Nox version can be specified in your ``noxfile.py`` file by simply defining
```

Comment:
nitpick: may make sense to drop the word 'simply' as there can be negative feelings about that verbiage in prose.

```suggestion
Minimum required Nox version can be specified in your ``noxfile.py`` file by defining
```